Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN yum update -y && yum install -y wget \
   && mvn package -Dmaven.test.skip=true \
   && mv target/perseo-core-*.war /usr/share/tomcat/webapps/perseo-core.war \
   && mkdir /var/log/perseo && chown tomcat:tomcat /var/log/perseo
+  # Force Tomcat to use ipv4 and optimize
+  && echo "JAVA_OPTS=\"-Djava.awt.headless=true -Xmx512m -XX:MaxPermSize=256m -XX:+UseConcMarkSweepGC -Djava.library.path=/usr/lib64:/usr/lib -Djava.security.egd=file:/dev/./urandom -Djava.net.preferIPv4Stack=true\"" >> /usr/share/tomcat/conf/tomcat.conf
```

Comment:
As far as I understand, IPv6 is one of the features of the IoT Platform that we want to keep...

Ok with all the other optimizations but, do we really need to remove IPv6? Is there any impediment in the on premise context that needs so?